scripts/common/chat.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Extends the Option to apply damage to targeted Enemy directly
+ * @param {HTMLElement} html 
+ * @param {Array} options 
+ * @returns 
+ */
+
+export const addChatMessageContextOptions = function (html, options) {
+    let canApply = li => {
+        const message = game.messages.get(li.data("messageId"));
+        return message.isRoll && message.isContentVisible && canvas.tokens?.controlled.length;
+    };
```

Comment:
Last comment I swear, but this doesn't seem to check whether it is in fact a damage roll right? I assume we don't want to  show the Apply Damage option for any and all messages.